src/test/java/uk/gov/hmcts/reform/et/syaapi/controllers/ManageCaseControllerTest.java

```diff
@@ -171,4 +189,53 @@ void shouldCreateDraftCase() throws Exception {
             .andExpect(jsonPath("$.created_date").exists())
             .andExpect(jsonPath("$.last_modified").exists());
     }
+
+    @Test
+    void shouldStartUpdateCase() throws Exception {
```

Comment:
Do we need to throws Exception here?

Reply:
Comments incorporated.

Reply:
Removed throws exception from Test suite.